_layouts/blog-post.html

```diff
@@ -0,0 +1,17 @@
+---
+layout: simple
+---
+
+<section class="py-5">
+  <div class="container text-center">
+    <h2>{{ page.title }}</h2>
+    <p>{{ page.printDate }}, by <a href="#">{{ page.author }}</a></p>
+    <img class="img-fluid my-5" src="https://bootstrapshuffle.com/placeholder/pictures/bg_16-9.svg" alt="">
```

Comment:
I think the image looks better the same width as the text column, so perhaps move this image element into the text column below.

I suspect `text-left` below is unnecessary.

Reply:
Done.